firmware/watchguard_sysa-dl.ksy

```diff
@@ -0,0 +1,139 @@
+meta:
+   id: wfsysadl
+   endian: be
+   title: Watchguard Fireware web update (Sysa-dl) file format
+   license: MIT
+
+
+seq:
+  - id: fwheader
+    type: fwheader
+    size: 24
+
+  - id: sections
+    type: sections
+    size: _io.size - 24
+
+
+types:
+  fwheader:
+     seq:
+         - id: md5sum
+           size: 0x10
+
+         - id: file_size
```

Comment:
Is this the file size of the compressed data that follows, or the uncompressed data? Depending on what it is you can probably use it to use as the size for the sections (in the top level `seq`), which is cleaner.

Reply:
file_size -- is whole size of file (with all sections)